.authors.yml

```diff
@@ -0,0 +1,391 @@
+- name: lorenz
+  email: looooo@users.noreply.github.com
+  num_commits: 1
+  first_commit: 2018-11-07 05:07:50
+  github: looooo
+- name: Maksim Rakitin
+  email: mrakitin@bnl.gov
+  alternate_emails:
+  - mrakitin@users.noreply.github.com
+  num_commits: 6
+  first_commit: 2019-07-19 17:37:24
+  github: mrakitin
+- name: Gonzalo Pena-Castellanos
+  email: goanpeca@gmail.com
+  num_commits: 7
+  first_commit: 2016-11-07 07:41:18
+  github: goanpeca
+- name: Matt McCormick
+  email: matt.mccormick@kitware.com
+  num_commits: 4
+  first_commit: 2019-07-31 17:05:02
+  github: thewtex
+- name: John Kirkham
+  email: jakirkham@gmail.com
+  alternate_emails:
+  - kirkhamj@janelia.hhmi.org
+  aliases:
+  - John Kirkham
+  - jakirkham
+  num_commits: 8
```

Comment:
Is this correct?

Reply:
hmmm, rever's commit counting seems to be off in some way.  Not sure what is going on here, but it is an issue with rever

Reply:
Note that this isn't used for anything other than author ordering

Reply:
WDYT about ordering authors by surname?